packages/engine/src/avatar/functions/moveAvatar.ts

```diff
@@ -147,23 +147,22 @@ const _additionalMovement = new Vector3()
  */
 
 const minimumDistanceSquared = 0.5 * 0.5
-
+const walkPoint = new Vector3() as Vector3
```

Comment:
unnecessary `as Vector3` cast

Reply:
No idea why I wrote the cast. But I added a commit specifically to remove that before I asked you to review so not sure why it still showed up for you.